cmake/dependencies.cmake

```diff
@@ -100,20 +100,15 @@ if(NOT GRPCXX_HERMETIC_BUILD)
 else()
     FetchContent_Declare(nghttp2
         URL      https://github.com/nghttp2/nghttp2/releases/download/v${NGHTTP2_MINVERSION}/nghttp2-${NGHTTP2_MINVERSION}.tar.xz
-        URL_HASH SHA256=19490b7c8c2ded1cf7c3e3a54ef4304e3a7876ae2d950d60a81d0dc6053be419
+        URL_HASH SHA256=88bb94c9e4fd1c499967f83dece36a78122af7d5fb40da2019c56b9ccc6eb9dd
     )
 
     set(ENABLE_LIB_ONLY   ON  CACHE BOOL "Build libnghttp2 only")
-    set(ENABLE_STATIC_LIB ON  CACHE BOOL "Build libnghttp2 in static mode")
-    set(ENABLE_SHARED_LIB OFF CACHE BOOL "Build libnghttp2 as a shared library")
+    set(BUILD_STATIC_LIBS ON  CACHE BOOL "Build libnghttp2 in static mode")
+    set(BUILD_SHARED_LIBS OFF CACHE BOOL "Build libnghttp2 as a shared library")
```

Comment:
If these values are set in the cache, since they are standard CMake variables, wouldn't they affect the build of grpcxx itself (i.e. force building a static rather than dynamic grpcxx.so?)

Reply:
🤦‍♂️ didn't realise these are standard cmake variables.